s3-pit-restore

```diff
@@ -125,9 +125,11 @@ class TestS3PitRestore(unittest.TestCase):
         self.assertNotEqual(bucket_versioning.status, None)
         print("enabled!")
 
-    def get_versions(self, s3, path, dest):
+    def get_versions(self, s3, path, dest, isDestPrefix):
```

Comment:
Can we use default arguments. so if we don't explicitly pass isDestPrefix it should be assumed false. `get_versions(self, s3, path, dest, isDestPrefix = False):`